app/views/gift_requests/gift_request.slim

```diff
@@ -1,34 +1,42 @@
 - action = @gift_request.new_record? ? "create" : "edit"
+- action_name = action == "create" ? "Create" : "Update"
```

Comment:
This should use the same conditional as the above line (`@gift_request.new_record?`) for clarity.